### PR TITLE
Update with new publications

### LINKED ISSUE #133: Add New Profiles
Add PCF, DSUBm, PDQm $match, BALP, and sIPS, 

new features in MHD

enhance mXDE and NPFS to new IG published specification.

Topologies whitepaper

### PR DESCRIPTION
**Pull request by JohnMoehrke**
close #133 

## 📑 Description

- [x] Not Completed
- [x] Completed

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed
- [x] I have selected a committee co-chair to review the PR

### CODE REVIEW

**Comment by JohnMoehrke** [comment]
Will review and improve during the May face-to-face meeting

**Comment by JohnMoehrke** [comment]
@jlamy I completely replaced the previous Provider Directory section with portions of the mCSD whitepaper and Topologies whitepaper. I really need you to help refine this. I might have pulled the right stuff, I might have not. There might be a better drawing, but I grabbed one simply because I think a drawing would be helpful. The intent is not to replace those whitepapers, but rather to inspire the reader of the HIE whitepaper to go read these others.

File: README.md
Comment (oliveregger) [inline README.md:1]: typos 3x challang -> challeng

**Review by oliveregger — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1141,17 +1161,18 @@ Where none of these conditions exist, then the choice could be [XDS](http://prof
...
+
 Each granularity level delivers unique benefits and this profile provides efficient access to both levels.
 
-This profile defines rules to ensure consistency and traceability of information used for clinical decisions. When a data element is accessed by a Clinical Data Consumer, identifiers from that data element can be used to access one or more documents in which this data element was originally recorded, providing a valuable broader clinical context.
+The ([mXDE](http://profiles.ihe.net/ITI/TF/Volume1/ch-45.html)) Profile defines rules to ensure consistency and traceability of information used for clinical decisions. When a data element is accessed by a Clinical Data Consumer. ([mXDE](http://profiles.ihe.net/ITI/TF/Volume1/ch-45.html)) Profile assures that all FHIR Resources made availalbe have Provenance linkage to the document(s) in which this data element was originally recorded, providing a valuable broader clinical context.
```
Comment (gpavan19) [inline README.md]: typos -> There is a dot (.) after "When a data element is accessed by a Clinical Data Consumer" that should be removed

**Review by gpavan19 — APPROVED** [review]

**Comment by JohnMoehrke** [comment]
update the version and release date. 
Add a hand written change section at the end of the text
From the top version text point at the hand written changes
Include link to github tracking

**Comment by JohnMoehrke** [comment]
will publish as 2.2

**Comment by matt-blackmon** [comment]
I read through this and think it is ready to move forward. I ddi not have really anything to add to the directory section.

**Pull request merged**